### DISCUSSION TITLE
Add ability to disable both "Move to adjacent display on repeated left or right commands" and "repeated half actions cycle 1/2, 2/3, and 1/3" options

### DISCUSSION THREAD

**Discussion by charleseff** [rxhanson/Rectangle#371] [Ideas]
It seems odd to have to choose between the two options of "Move to adjacent display on repeated left or right commands" and "repeated half actions cycle 1/2, 2/3, and 1/3" options".  I'd rather that entering a rectangle command more than once does nothing.

**Comment by rxhanson** [comment]
https://github.com/rxhanson/Rectangle#halves-to-thirds-repeated-execution-of-half-and-quarter-actions

Changing to a discussion so users can upvote if they want this in the UI

**Reply by charleseff** [reply]
Some rationale: I may focus a window that's toward the right side of the screen, then hit the "Right half" shortcut just to ensure it's at exactly half.  If it were already at exactly half, I'd want the command to do nothing, which would be great.  Instead, it sends the window to the other screen, which is a jarring experience for me.  This is one example of why I think it's valuable for these shortcuts to represent "end states" of windows

**Comment by charleseff** [comment]
`defaults write com.knollsoft.Rectangle subsequentExecutionMode -int 2`
does what i want so i don't think we need this anymore

**Reply by rxhanson** [reply]
I don't mind leaving it up, just to gauge interest on things that make sense to add into the UI (or for other people that miss that segment in the Readme and go down this path).
